translation/google.py

```diff
@@ -29,7 +29,7 @@ def speech_to_text(self, audio_file: Any, input_language: str):
         config = speech.RecognitionConfig(
             encoding=speech.RecognitionConfig.AudioEncoding.LINEAR16,
             sample_rate_hertz=16000,
-            language_code=input_language,
+            language_code="en-US",
```

Comment:
Seems to be switching to a hardcoded value instead of picking up from a variable. Is that intentional ?